### ISSUE TITLE
[ppc] misleading indentation warnings (Trac #1658)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1650]
```text
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx: In member function ‘void xppc::ini::set()’:
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:294:4: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
    if(s.dl>dz) s.dl=dz; if(s.dh<dz) s.dh=dz;
    ^~
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:294:25: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘if’
    if(s.dl>dz) s.dl=dz; if(s.dh<dz) s.dh=dz;
                         ^~
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:576:4: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
    if(flag) flag=(bool)(inFile >> B >> Be); fail=!flag;
    ^~
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:576:45: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘if’
    if(flag) flag=(bool)(inFile >> B >> Be); fail=!flag;
                                             ^~~~
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:577:4: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
    if(flag) flag=(bool)(inFile >> D >> De); if(!flag) D=pow(wv0, k);
    ^~
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:577:45: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘if’
    if(flag) flag=(bool)(inFile >> D >> De); if(!flag) D=pow(wv0, k);
                                             ^~
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:578:4: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
    if(flag) flag=(bool)(inFile >> E >> Ee); if(!flag) E=0;
    ^~
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:578:45: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘if’
    if(flag) flag=(bool)(inFile >> E >> Ee); if(!flag) E=0;
                                             ^~
[  0%] Built target clsim-make_safeprimes
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ppc.cxx: In function ‘int xppc::flset(int, int)’:
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ppc.cxx:591:7: warning: this ‘for’ clause does not guard... [-Wmisleading-indentation]
       for(int m=0; m<3; m++) r[m]=q.oms[n].r[m]; break;
       ^~~
/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ppc.cxx:591:50: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘for’
       for(int m=0; m<3; m++) r[m]=q.oms[n].r[m]; break;
                                                  ^~~~~
```

Migrated from https://code.icecube.wisc.edu/ticket/1658
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "{{{\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx: In member function \u2018void xppc::ini::set()\u2019:\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:294:4: warning: this \u2018if\u2019 clause does not guard... [-Wmisleading-indentation]\n    if(s.dl>dz) s.dl=dz; if(s.dh<dz) s.dh=dz;\n    ^~\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:294:25: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018if\u2019\n    if(s.dl>dz) s.dl=dz; if(s.dh<dz) s.dh=dz;\n                         ^~\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:576:4: warning: this \u2018if\u2019 clause does not guard... [-Wmisleading-indentation]\n    if(flag) flag=(bool)(inFile >> B >> Be); fail=!flag;\n    ^~\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:576:45: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018if\u2019\n    if(flag) flag=(bool)(inFile >> B >> Be); fail=!flag;\n                                             ^~~~\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:577:4: warning: this \u2018if\u2019 clause does not guard... [-Wmisleading-indentation]\n    if(flag) flag=(bool)(inFile >> D >> De); if(!flag) D=pow(wv0, k);\n    ^~\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:577:45: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018if\u2019\n    if(flag) flag=(bool)(inFile >> D >> De); if(!flag) D=pow(wv0, k);\n                                             ^~\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:578:4: warning: this \u2018if\u2019 clause does not guard... [-Wmisleading-indentation]\n    if(flag) flag=(bool)(inFile >> E >> Ee); if(!flag) E=0;\n    ^~\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ini.cxx:578:45: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018if\u2019\n    if(flag) flag=(bool)(inFile >> E >> Ee); if(!flag) E=0;\n                                             ^~\n[  0%] Built target clsim-make_safeprimes\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ppc.cxx: In function \u2018int xppc::flset(int, int)\u2019:\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ppc.cxx:591:7: warning: this \u2018for\u2019 clause does not guard... [-Wmisleading-indentation]\n       for(int m=0; m<3; m++) r[m]=q.oms[n].r[m]; break;\n       ^~~\n/home/dschultz/Documents/combo/trunk/src/ppc/private/ppc/ocl/ppc.cxx:591:50: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018for\u2019\n       for(int m=0; m<3; m++) r[m]=q.oms[n].r[m]; break;\n                                                  ^~~~~\n}}}", 
    "reporter": "david.schultz", 
    "cc": "olivas", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "[ppc] misleading indentation warnings", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2016-04-26T19:52:51", 
    "milestone": "", 
    "owner": "dima", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161116T10:20:38: 
* kjmeagher commented:

> In changeset 151536:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="151536"
> fix horrible formatting, closes #1650
> ```


* kjmeagher changed status from "new" to "closed"
* kjmeagher changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas commented:

> Milestone IceSim 5.1 deleted


* olivas changed milestone from "IceSim 5.1" to ""
